modules/explorer/info.go

```diff
@@ -38,6 +42,32 @@ func (e *Explorer) BlockFacts(height types.BlockHeight) (modules.BlockFacts, boo
 	return bf.BlockFacts, true
 }
 
+// LatestBlockFacts returns a set of statistics about the blockchain as they appeared
+// at the latest block height in the explorer's consensus set.
+func (e *Explorer) LatestBlockFacts() modules.BlockFacts {
+	var bf blockFacts
```

Comment:
The explorer does not have its own mutex. It doesn't store anything in memory at all. bolt's internal locking is sufficient. This may change if we later decide to cache things in memory.